_drafts/2016-12-21-commcallv12-review-coc.md

```diff
@@ -0,0 +1,84 @@
+---
+name: commcallv12-review-coc
+layout: post
+title: "Highlights and Resources from Community Call v12: How do I create a code of conduct for my event/lab/codebase?"
+date: 2016-12-21
+authors:
+  - name: Stefanie Butland
+categories:
+  - blog
+tags:
+- ropensci
+- community
+- events
+
+---
+Our [Community Call](https://ropensci.org/blog/blog/2016/10/31/comm-call-v12) on December 15th covered a big topic in tech communities: "How do I create a code of conduct for my event/lab/codebase?". Here, we cover some of the key themes and considerations that arose from the discussion and point to curated resources and examples to follow when developing a code of conduct (CoC) for your community.
+
+Three guest speakers shared different perspectives. Dr Pauline Barmby talked about the process and lessons learned as Data and Software Carpentries recently updated their CoC; Ms Safia Abdalla talked about "Codes of conduct for open source: the stuff no one tells you"; and Dr Titus Brown talked about his lab CoC. You can [read their bios](#speaker-bios) below and watch the [video](https://vimeo.com/196503807) of the call. Speakers did not use other presentation materials.
+
+From the talks and discussion it became clear that there are three key components to codes of conduct:
+
+1. drafting
+2. adopting and sharing
+3. reporting and enforcing
+
+## Drafting a Code of Conduct
+
+Once a group of interested people digs into the work of drafting a CoC, the process can seem like it's going on forever, especially if you try to incorporate "everyone's" feedback. The CoC cannot be a living document. It's most important to produce something good and perhaps revisit it each year. There seemed to be a consensus that a small committee of key stakeholders should draft the CoC. For his own lab, Titus drafted and posted it without fanfare, but has already seen [unanticipated benefits](http://ivory.idyll.org/blog/2016-notes-on-lab-coc.html).
+
+Some notes:
+
+* Develop and post a CoC whether or not you know of any problems; setting clear expectations is 90% of the work
+* The CoC has to set expectations for everyone, including the boss
+* One big issue for the Carpentries was whether or not there should be specific lists of prohibited behaviors or protected groups
+* Because many of our communities cross international boundaries, our CoCs should be accessible in multiple languages. The [Contributor Covenant](http://contributor-covenant.org/) does provide a template CoC in [multiple languages](http://contributor-covenant.org/i18n/).
```

Comment:
one point to include might be that rules differ by country, something to keep in mind